tests/cli/workflow/conftest.py

```diff
@@ -61,6 +61,47 @@ def fixture_analysis_store(
     return _store
 
 
+@pytest.fixture(name="fastq_context")
+def fixture_fastq_context(
+    base_context,
+    cg_context: CGConfig,
+    fastq_case,
+    helpers: StoreHelpers,
+    case_id,
+) -> CGConfig:
+    """Returns a CGConfig where the meta_apis["analysis_api"] is a FastqAnalysisAPI and a store
+    containing a fastq case"""
+    _store = cg_context.status_db
+    # Add fastq case to db
+    fastq_case["samples"][0]["sequenced_at"] = datetime.now()
+    helpers.ensure_case_from_dict(store=_store, case_info=fastq_case)
+    return cg_context
+
+
+@pytest.fixture(name="fastq_case")
+def fixture_fastq_case(case_id, family_name, sample_id, cust_sample_id, ticket_nr) -> dict:
+    """Returns a dict describing a fastq case"""
+    return {
+        "name": family_name,
+        "panels": None,
+        "internal_id": case_id,
+        "data_analysis": "fastq",
+        "data_delivery": "fastq",
```

Comment:
Are these defined as constants somewhere? It would be nice that the fixture also changes if the constants change

Reply:
I can't find any. Do you suggest making a fixture that just returns the DataDelivery and Pipeline class respectively and use that?

Reply:
Or, why can't I just import them and use them in the pytest?

Reply:
Yes, exactly that! Import them directly and use them